packages/twenty-server/nest-cli.json

```diff
@@ -4,7 +4,7 @@
   "sourceRoot": "src",
   "compilerOptions": {
     "builder": "swc",
-    "typeCheck": true,
+    "typeCheck": false,
```

Comment:
I would check if we can deactivate it with an arg passed to the relevant commands instead of in the config file for everything.

Reply:
Yes it is possible https://docs.nestjs.com/recipes/swc#type-checking